autonity_cli/__main__.py

```diff
@@ -22,7 +23,9 @@
 
 @group(context_settings=dict(help_option_names=["-h", "--help"]))
 @option("--verbose", "-v", is_flag=True, help="Enable additional output (to stderr)")
-@version_option()
+@version_option(
+    message=f"%(prog)s, version %(version)s\nprotocol version {protocol_version}"
```

Comment:
I think it's better to condense into one line as follows:

```suggestion
    message=f"Autonity CLI v%(version)s (Autonity Protocol {protocol_version})"
```